korgan/legal_calc.py

```
@@ -36,29 +89,81 @@
-_IIN_RE = re.compile(r"(?<!\d)\d{12}(?!\d)")
+_IIN_LABELED_RE = re.compile(r"\bИИН\s*[:\-–]?\s*(\d{12})\b", re.IGNORECASE)
+_BIN_LABELED_RE = re.compile(r"\bБИН\s*[:\-–]?\s*(\d{12})\b", re.IGNORECASE)
+_BARE_12_DIGITS_RE = re.compile(r"(?<!\d)\d{12}(?!\d)")
+_IIN_RE = _BARE_12_DIGITS_RE
+
+
+def _round_tenge(value: Decimal) -> int:
+    return int(value.quantize(Decimal("1"), rounding=ROUND_HALF_UP))
 
 
 def calc_gosposhlina_claim(amount: int, is_individual: bool) -> int:
```

Ruff (0.16.2)

[warning] Boolean-typed positional argument in function definition

(FBT001)